proto/buf/alpha/registry/v1alpha1/schema.proto

```diff
@@ -21,20 +21,33 @@ import "google/protobuf/descriptor.proto";
 
 // Provides endpoints for downloading dynamic schemas and for using schemas
 // in validation and data transformation functions.
-service SchemaService {
-  // GetSchema allows the caller to download a schema for one or more requested
+// TODO don't like name
+service FileDescriptorSetService {
```

Comment:
```suggestion
service DescriptorService {
```

Reply:
some nuance here...better name but the problem is that a `DescriptorProto` should be named a `MessageDescriptorProto` but isn't https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/descriptor.proto#L98